### PR TITLE
Add shoot-rsyslog-relp extension to the list of extensions

### PR DESCRIPTION
**Pull request by ialidzhikov**
**What this PR does / why we need it**:
Add shoot-rsyslog-relp extension to the list of extensions.

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:
N/A

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: .docforge/documentation/gardener-extensions/others.yaml [hunk 1]
```diff
@@ -55,3 +55,11 @@ structure:
       description: Gardener extension controller which deploys pull-through caches for container registries.
     source: https://github.com/gardener/gardener-extension-registry-cache/blob/DEFAULT_BRANCH/README.md
   - fileTree: https://github.com/gardener/gardener-extension-registry-cache/tree/main/docs
+- dir: gardener-extension-shoot-rsyslog-relp
+  structure:
+  - file: _index.md
+    frontmatter:
+      title: Node Audit Logging
+      description: Gardener extension controller which configures the rsyslog an auditd services installed on shoot nodes.
```
Comment (JordanJordanov) [inline .docforge/documentation/gardener-extensions/others.yaml:63]:
```suggestion
      description: Gardener extension controller which configures the rsyslog and auditd services in the shoot nodes.
```

Comment (ialidzhikov) [reply .docforge/documentation/gardener-extensions/others.yaml:63]: I copy-pasted from https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/c09737d2ab8780c87463e5a685cc2ed89988a5e8/README.md?plain=1#L6 🙄 

Let me first fix in the README.md of the shoot-rsyslog-relp extension.

**Review by JordanJordanov — APPROVED** [review]
/lgtm

**Review by n-boshnakov — APPROVED** [review]
/lgtm

**Pull request merged**